lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -163,5 +166,16 @@ sub get_encrypted_volume_activation {
     return Installation::SystemProbing::EncryptedVolumeActivationController->new();
 }
 
+sub get_disk_activation_selector {
```

Comment:
s/get_disk_activation_selector/get_disk_activation/ It is a controller not a selector or other thing, and we agreed to not use _controllers as from $testapi::distri-> all the calls we do from this layer is to Controllers' layer.